### PR TITLE
Fix Mariadb_address in new documentation

### PR DESCRIPTION
**Pull request by holser**
There were doc update after PR#458. This is follow up update to update this part of documentation.

### CODE REVIEW

File: docs_user/modules/proc_migrating-databases-to-mariadb-instances.adoc [hunk 1]
```diff
@@ -116,7 +116,7 @@ oc wait --for condition=Ready pod/mariadb-copy-data --timeout=30s
 
 .Procedure
 
-. Check that the Galera database cluster members are online and synced:
+. Check that the Source galera database cluster members are online and synced:
```
Comment (jistr) [inline docs_user/modules/proc_migrating-databases-to-mariadb-instances.adoc:119]: I think this should be capitalized differently: "source Galera". It's a small thing but now that these docs directly translate to customer docs, we need to care about such stuff :).

Comment (holser) [reply docs_user/modules/proc_migrating-databases-to-mariadb-instances.adoc]: updated in next update

**Review by jistr — APPROVED** [review]
:+1: Thanks

**Comment by jistr** [comment]
recheck

**Pull request merged**